main.tf

```diff
@@ -34,6 +34,16 @@ resource "aws_security_group" "runner" {
   tags = "${local.tags}"
 }
 
+resource "aws_security_group_rule" "ssh_from_cidr_blocks" {
```

Comment:
in case you need ssh access to runner you just add your host to the same sg where runner is located, by attaching to https://github.com/lean-delivery/tf-module-aws-gitlab-runner/blob/master/outputs.tf#L6